main.tf

```diff
@@ -0,0 +1,71 @@
+provider "github" {}
+
+locals {
+  pr_teams   = "${length(var.restricted_pr_teams) != 0 ? var.restricted_pr_teams : list("")}"
```

Comment:
Make sure you are using terraform 0.11.14... the ternary operator cannot return lists until tf 0.12...

Reply:
Ok, so what is the 0.11.14 equivalent of doing this?

Reply:
Not really sure I understand why this is a thing, but maybe just set the default to `[""]`?

Reply:
This has been fixed. I managed to overcomplicate it when I was previously having difficulty with getting terraform to recognize the variable as a list and therefore had to put the variable inside brackets `[]`. Anyway, now it seems to work fine. Also downgraded my version of terraform to ensure it works correctly.